DirectEmployers/MyJobs
# Close calendar when "Apply" is clicked

## What's this PR do?
Close calendar when "Apply" is clicked
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
In Partner Assessment list, open calendar and click apply. The calendar should close
## Any background context you want to provide?
